### DISCUSSION TITLE
Version 2.8.7 VDL Mode 2 support now available.

### DISCUSSION THREAD

**Discussion by jprochazka** [jprochazka/adsb-receiver#608] [Announcements]
### One of the biggest feature releases yet!

3 decoders options plus 2 extras in a single release.

This evening, I merged the vldm2 branch into master completing the task of adding VDL Mode 2 support to the project. Two VDL Mode 2 decoder options are now included which are capable of installing either [VDLM2DEC](https://github.com/TLeconte/vdlm2dec) or [dumpvdl2](https://github.com/szpajder/dumpvdl2). It should be noted that at this time the old PHP portal only shows VDLM2DEC via the ACARS page due to the fact that VDLM2DEC saves message data in the same format and same database as ACARSDEC does through ACARSSERV. However, dumpvdl2 message data is saved to a separate SQLite database in a different format. Although not yet available, look for a way to view dumpvdl2 messages via the existing PHP portal to be added shortly.

Along with VDL Mode 2 support the ability to choose [Readsb](https://github.com/wiedehopf/readsb) as an ADS-B decoder has been added to the installer. Options to install [Graphs1090](https://github.com/wiedehopf/graphs1090) and [tar1090](https://github.com/wiedehopf/tar1090) have been added to the extras list as well.

Other noticeable changes include an improved device assignment process when installing multiple decoders, new decoder selection menus, and Noble bug fixes.

Thanks goes out to the following people who helped make this release possible via their contributions and input. 

- @MCancellier and @thebaldgeek for their input and suggestions pertaining to ACARS and VDL Mode 2 support.
- @wiedehopf for his help and continued maintenance of the [Readsb](https://github.com/wiedehopf/readsb), [Graphs1090](https://github.com/wiedehopf/graphs1090) and [tar1090](https://github.com/wiedehopf/tar1090) projects.

Now, back to ADS-B Portal backend development so v2.9.0 can be released before v2.8.x patch versions hit the double digits. In the meantime, don't be afraid to point out any issues or additions you feel should come ahead of v2.9.0. Look for an ADS-B Portal update to be posted soon!

**Comment by Tom-Neverwinter** [comment]
Hype! Thank you for all the work!!!

**Reply by jprochazka** [reply]
Not a problem. It has been fun getting things back in order and adding new features.
